### PR TITLE
added magma hardfork history

### PR DESCRIPTION
**Pull request by kjeom**
## Proposed changes

- Added magma hard fork history

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- #475

### CODE REVIEW

File: docs/klaytn-history/README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 
+## Magma
+| ` ` | Baobab | Cypress |
+| --------------- | --------------- | --------------- |
+| Date | Aug 08, 2022 11:01:20 / UTC+9 | Aug 29, 2022 11:51:00 / UTC+9 |
+| Block number  | `#98,347,376`  | `#99,841,497` |
+
+### Summary
+
+Magma hardfork were introduced with the [v1.9.0 release](https://github.com/klaytn/klaytn/releases/tag/v1.9.0). It includes dynamic gas fee pricing mechanism, [#1493](https://github.com/klaytn/klaytn/pull/1493))
```
Comment (kjhman21) [inline docs/klaytn-history/README.md:13]: @kjeom It would be good to have a link of KIP-71 spec (https://kips.klaytn.foundation/KIPs/kip-71)

Comment (kjeom) [reply docs/klaytn-history/README.md]: Reflected, Thanks

File: docs/klaytn-history/README.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
 
+## Magma
+| ` ` | Baobab | Cypress |
+| --------------- | --------------- | --------------- |
+| Date | Aug 08, 2022 11:01:20 / UTC+9 | Aug 29, 2022 11:51:00 / UTC+9 |
+| Block number  | `#98,347,376`  | `#99,841,497` |
+
+### Summary
+
+Magma hardfork were introduced with the [v1.9.0 release](https://github.com/klaytn/klaytn/releases/tag/v1.9.0). It includes dynamic gas fee pricing mechanism, [#1493](https://github.com/klaytn/klaytn/pull/1493)) and is an implementation of [KIP-71](https://kips.klaytn.foundation/KIPs/kip-71)
```
Comment (terrikwak) [inline docs/klaytn-history/README.md:13]:
```suggestion
Magma hardfork was introduced with the [v1.9.0 release](https://github.com/klaytn/klaytn/releases/tag/v1.9.0). It includes dynamic gas fee pricing mechanism, [#1493](https://github.com/klaytn/klaytn/pull/1493)) and is an implementation of [KIP-71](https://kips.klaytn.foundation/KIPs/kip-71)
```

**Review by hyunsooda — APPROVED** [review]
LGTM. Please close #475 as well.

**Pull request merged**